developer_guide/migration/index.rst

```diff
@@ -96,11 +96,12 @@ Here are the migration guides:
 
 **Enterprise Edition**
 
-Please contact our team through your access to the support service desk to ask for a standard archive.
+Standard EE archives are now available on a dedicated portal, pleae contact your sales manager or channel partner to have access to this portal if it's not already the case. You will be able to download your EE archive directly from the Portal interface.
```

Comment:
"channel partner" or "channel manager"?
"to have access to" could be "to get access to"
"to this portal" could be "to it"
"if it's" should be "if it is". I would suggest not to use contractions on docs.